api/courses.go

```diff
@@ -303,13 +291,16 @@ func sortCourses(courses []model.Course) {
 }
 
 func (r coursesRoutes) getCourseBySlug(c *gin.Context) {
```

Comment:
I've only noticed that I can't view public courses anymore when not logged-in.
This appears to be introduced by this PR if I am not mistaken (works on dev).
E.g., I can see https://1114.test.live.mm.rbg.tum.de/?year=2022&term=W&slug=t2&view=3, but not https://1095.test.live.mm.rbg.tum.de/. Seems like a permission issue which keeps it from loading at all. 🤔 (Happens both on Chrome and Firefox.) Maybe related to these changes here?

Reply:
Not related to these changes but still good catch! Thanks for pointing out. Should be fixed now.

Reply:
Okay, just was curious. Thanks for the quick fix, works now again!